PyZAPI/production.ini.example

```diff
@@ -0,0 +1,59 @@
+###
```

Comment:
I would think they would be different, esp your database connection details would be different